charts/fleet/templates/ingress.yaml

```diff
@@ -28,9 +32,17 @@ spec:
     http:
       paths:
       - path: /*
-        # Next line required in k8s 1.19 and not supported in <=1.17
-        # pathType: Exact
+      # Next block required in k8s =>1.19 and not supported in <=1.17
+      {{- if semverCompare ">=1.19.0-0" .Capabilities.KubeVersion.GitVersion }}
+        pathType: Exact
+        backend:
+          service:
+            name: fleet
+            port:
+              number: fleet
```

Comment:
Should this be number or name?

Reply:
https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.19/#servicebackendport-v1-networking-k8s-io

Reply:
Missed that one, fixed!